package.json

```diff
@@ -53,7 +53,7 @@
     "json-loader": "0.5.7",
     "klokantech-noto-sans": "git+https://github.com/QwantResearch/fonts.git#f25a802",
     "mapbox-gl": "0.49.0",
-    "mapbox-gl-js-mock": "https://github.com/QwantResearch/mapbox-gl-js-mock.git#58d80b5",
+    "mapbox-gl-js-mock": "https://github.com/QwantResearch/mapbox-gl-js-mock.git#QMAPS-534_scale_control",
```

Comment:
don't forget to merge the PRs on 'mapbox-gl-js-mock.git' and fix the right commit on master once it's merged

Reply:
sure